src/appleseed.studio/python/module.cpp

```diff
@@ -135,4 +136,6 @@ BOOST_PYTHON_MODULE(_appleseedstudio)
 
     bpy::def("main_window", main_window_as_pylong);
     bpy::def("create_dock_widget", create_dock_widget, bpy::args("dock_name"));
+
+    boost::python::def("get_studio_dir", get_executable_directory);
```

Comment:
Then I think what you really want is `appleseed::shared::Application::get_root_path()`.